packages/purpose-process/src/services/purposeService.ts

```diff
@@ -874,6 +877,29 @@ export function purposeServiceBuilder(
         );
       }
 
+      return riskAnalysisFormConfig;
+    },
+    async retrieveLatestRiskAnalysisConfiguration({
+      tenantKind,
+      organizationId,
+    }: {
+      tenantKind: TenantKind | undefined;
+      organizationId: TenantId;
+    }): Promise<RiskAnalysisFormRules> {
+      logger.info(`Retrieve latest risk analysis configuration`);
+
+      const tenant = await retrieveTenant(organizationId, readModelService);
+      const kind = tenantKind || tenant.kind;
+
+      if (!kind) {
+        throw tenantKindNotFound(tenant.id);
+      }
```

Comment:
We could skip a potentially useless read model query this way:
```suggestion
      async function retrieveTenantKindFromTenant(): Promise<TenantKind> {
        const tenant = await retrieveTenant(organizationId, readModelService);
        assertTenantKindExists(tenant);
        return tenant.kind;
      }

      const kind = tenantKind || (await retrieveTenantKindFromTenant());
```